### PR TITLE
[EBE-5521] Use JWT instead of old UUID token in Authorization header

### PR DESCRIPTION
**Pull request by jhita**
Use the new `Authorization` header instead of the old `ElasticBox-Token` header, and also the new JWT and not the old UUID tokens.

@EfrenRey please review.

### CODE REVIEW

File: Application Lifecycle Management/api-blobs.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Uploads a file using multi-part form data when you give these parameters in the
 Headers:
 
 Content-Type: application/json
-Elasticbox-Token: your_authentication_token
+Authorization: your_json_web_token
```
Comment (EfrenRey) [inline Application Lifecycle Management/api-blobs.md]: This should be ```Authorization: Bearer <your_json_web_token>```

File: Examples/deploy-a-box.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ password=$2
 All API calls start with signing in to the Cloud Application Manager website and [getting an authentication token](../Getting Started/api-overview-and-access.md). You use this token to perform tasks in your Cloud Application Manager workflow. In this example, we pass the token in the format as shown to all of the API requests that relate to deploying MongoDB.
 
 ```
-ElasticBox-Token:8ccc8203-2efd-44a9-8819-e95fd2277be2
+Authorization:eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJqb3NlaWduYWNpbyIsImp0aSI6IjA0YWVlNGQwLTBiNDQtNGI4Ny05MjkxLTFjZmU4MzQ4YWU4MCIsImV4cCI6MTUxMTg5MTc5Miwib3JnYW5pemF0aW9uIjoiZWxhc3RpY2JveCIsImlhdCI6MTUxMDU5NTc5MiwidHlwZSI6InVzZXIiLCJuYW1lIjoiSm9zZSBJZ25hY2lvIEhpdGEgQCBDQU0ifQ.iUXcLPI41rd9ngcl-mveY5hcrS9N38zAJ85lTfcJqkevDZAN92p8zj-ZGURijUQDAiOi7OBqaLk45K3ba_m1s7cjaZfvBcnmI2Ig2BPyaOUXuvUW35H6SfOPNsoq1kvqgwFWHNx-xkX_qTmFHvEfLmQUhol5cTh7jkB_lRTRzwAf7UsiEp6mZmBuL5ERcxZeWLmfh8tA-kwaWTvlUPtD4CyX4mFeu-7u4B59dcivG4PJ8kmO7pTqm5vTnfb6JWSaHrQsAL-wCLjnWOwDlEmgqL43_jrq8GiwiduiC4qXVZdbwjLhXWGhIpyIVPuXZREnU1EZNhr2Gfj_F4ujgKdxww
```
Comment (EfrenRey) [inline Examples/deploy-a-box.md]: ```Authorization:Bearer eyJhbGciOiJSUzI1NiIsI....```

Also, please don't use a valid token.

File: Examples/deploy-a-box.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ payload="{
 policy_box=$(curl -k -s \
         -X POST \
         -H "Content-Type:application/json" \
-        -H "elasticBox-token:$elasticBox_token" \
+        -H "Authorization:$json_web_token" \
```
Comment (EfrenRey) [inline Examples/deploy-a-box.md]: ```        -H "Authorization:Bearer $json_web_token" \```

File: Getting Started/api-overview-and-access.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ Headers for a sample request look like this.
 Headers:
 
 Content-Type: application/json
-ElasticBox-Token: 46-cbd2-4273-85b8-e2bba8ed265a
+Authorization: eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJqb3NlaWduYWNpbyIsImp0aSI6IjA0YWVlNGQwLTBiNDQtNGI4Ny05MjkxLTFjZmU4MzQ4YWU4MCIsImV4cCI6MTUxMTg5MTc5Miwib3JnYW5pemF0aW9uIjoiZWxhc3RpY2JveCIsImlhdCI6MTUxMDU5NTc5MiwidHlwZSI6InVzZXIiLCJuYW1lIjoiSm9zZSBJZ25hY2lvIEhpdGEgQCBDQU0ifQ.iUXcLPI41rd9ngcl-mveY5hcrS9N38zAJ85lTfcJqkevDZAN92p8zj-ZGURijUQDAiOi7OBqaLk45K3ba_m1s7cjaZfvBcnmI2Ig2BPyaOUXuvUW35H6SfOPNsoq1kvqgwFWHNx-xkX_qTmFHvEfLmQUhol5cTh7jkB_lRTRzwAf7UsiEp6mZmBuL5ERcxZeWLmfh8tA-kwaWTvlUPtD4CyX4mFeu-7u4B59dcivG4PJ8kmO7pTqm5vTnfb6JWSaHrQsAL-wCLjnWOwDlEmgqL43_jrq8GiwiduiC4qXVZdbwjLhXWGhIpyIVPuXZREnU1EZNhr2Gfj_F4ujgKdxww
```
Comment (EfrenRey) [inline Getting Started/api-overview-and-access.md]: ```Authorization:Bearer eyJhbGciOiJSUzI1N....```

**Comment by jim-phillips** [comment]
@jhita Please create a PR against the cam branch.  I am closing this PR

**Pull request closed without merging**